docs/changelog/0.4.0.md

```diff
@@ -0,0 +1,532 @@
+
+# 0.4.0 (Jan 14, 2021)
+
+We are happy to announce the next major release of Scala Native!
+
+Release 0.4.0 brings many changes to Scala Native ecosystem, let's take a look at few of them. 
+
+## Compiler changes
+### Supported Scala version
+The most important change introduced by this release is support for recent Scala versions, that means you're able to work
+with the most recent versions of Scala 2 available at the moment of release - 2.12.12 and 2.13.4. 
+Scala 2.11 is still support, what more it was enhanced by multiple features, i.e. handling of java default methods.
+
+### JUnit Native Plugin
+Scala Native now comes with JUnit support out of the box, this means that you can write tests in the same way you would do for a Java project.
+To enable JUnit tests all you will need to do is to add the two following lines to your `build.sbt`.
+
+```scala
+addCompilerPlugin("org.scala-native" % "junit-plugin" % SNVersion cross CrossVersion.full)
+```
+
+### NIR changes
```

Comment:
NIR changes means nothing to most users. There are two pieces of important information:
1. This release is not binary compatible with previous releases. That should be the very first piece of information the Breaking changes section, or even in the intro.
2. There are some user-visible bugs that have been fixed. We should talk about the user-visible stuff in the Bug fixes section.